packages/preview-middleware-client/src/utils/version.ts

```diff
@@ -55,16 +57,33 @@ export function isLowerThanMinimalUi5Version(
         if (ui5VersionInfo.major < minUi5VersionInfo.major) {
             return true;
         }
-        if (
-            ui5VersionInfo.major === minUi5VersionInfo.major &&
-            ui5VersionInfo.minor < minUi5VersionInfo.minor
-        ) {
+        if (ui5VersionInfo.major === minUi5VersionInfo.major && ui5VersionInfo.minor < minUi5VersionInfo.minor) {
             return true;
         }
     }
     return false;
 }
 
+/**
+ * Checks if the given version is equal to the specified version.
+ * @param ui5VersionInfo to check
+ * @param targetUi5VersionInfo to check against (default is 1.71)
+ *
+ * @returns boolean
+ */
+export function isEqualToUi5Version(
```

Comment:
The function name is misleading. It is not checking version equality. I'd suggest something like isVersionPatchLowerThan with according code logic adaptation ((ui5VersionInfo?.patch ?? 0) < (targetUi5VersionInfo?.patch ?? 0));

Reply:
`isVersionEqualOrHasNewerPatch` is more close. I adapted.